embedding-stores/langchain4j-community-sqlserver/src/main/java/dev/langchain4j/store/embedding/sqlserver/util/EmbeddingStoreUtil.java

```diff
@@ -0,0 +1,115 @@
+package dev.langchain4j.store.embedding.sqlserver.util;
+
+import com.fasterxml.jackson.core.json.JsonWriteFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dev.langchain4j.data.document.Metadata;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import microsoft.sql.Vector;
+
+/**
+ * Utility methods for working with SQL Server embedding stores.
+ */
+public class EmbeddingStoreUtil {
```

Comment:
```suggestion
class SQLServerEmbeddingStoreUtil {
```

Would be better if make it package-private for internel use.